lib/index.js

```diff
@@ -7,7 +7,7 @@ const loglevel = argv.loglevel || 'info';
 const logger = getLogger(loglevel);
 const options = { loglevel: loglevel };
 
-runner(argv.webpack, argv.dependency, options)
+runner(argv.webpack, argv.dependency, options, argv.packageManager)
```

Comment:
I would place this in `options`. With the addition of #23, this is going to get complicated fast.